src/dotnet/APIView/APIViewWeb/Helpers/AzureEngSemanticVersion.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+// Ported from https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/SemVer.ps1
+public class AzureEngSemanticVersion : IComparable<AzureEngSemanticVersion>
```

Comment:
Do we really need semantic version handler or just use the version coming in codefile?

Reply:
We are using codeFile version, but need a way to parse it. This seems to be the most reliable for all languages.